common/arrow/Cargo.toml

```diff
@@ -14,9 +14,9 @@ simd = ["arrow/simd"]
 
 # Github dependencies
 
-arrow = { git="https://github.com/datafuse-extras/arrow-rs", rev = "ed61617" }
-arrow-flight = { git="https://github.com/datafuse-extras/arrow-rs", rev = "ed61617" }
-parquet = { git="https://github.com/datafuse-extras/arrow-rs", rev = "ed61617", features = ["arrow"] }
+arrow = { git="https://github.com/zhaox1n/arrow-rs", branch = "refine_aggregate_funtion" }
```

Comment:
Really nice.
https://github.com/datafuse-extras/arrow-rs is ready for this patch.